nx-X11/programs/Xserver/Xext/panoramiX.c

```diff
@@ -149,7 +150,7 @@ GCFuncs XineramaGCFuncs = {
 
 
 static Bool
-XineramaCloseScreen (ScreenPtr pScreen)
+XineramaCloseScreen (int i, ScreenPtr pScreen)
```

Comment:
CloseScreen API is already more advanced. Drop ``int i`` here again and keep this line as is.

Reply:
done